Dockerfile

```diff
@@ -1,3 +1,4 @@
 FROM ruby:2.6
+RUN gem install bump-cli
```

Comment:
Gem installation is done directly in the `entrypoint.sh` file ([see here](https://github.com/bump-sh/github-action/blob/master/entrypoint.sh#L4)), we shouldn't need to add it here.

Reply:
It's not a duplicate, I removed it from the entrypoint.sh, it's just a move to the expected location

If you really prefer not placing deps install in the Dockerfile, I can move it back to the entrypoint.sh and add a comment pointing to this issue for other people that don't understand why it was done in that way :) up to you

Reply:
Well, moving the `gem install` at the `Dockerfile` level seems to change the behavior: we now get Ruby version issues, like you can see here: https://github.com/bump-sh/bump-ci-example/runs/840628727?check_suite_focus=true

I did the same test but keeping the `gem install` inside the `entrypoint.sh` script, and this time it worked: https://github.com/bump-sh/bump-ci-example/runs/840555890?check_suite_focus=true

I don't know what's happening, but I propose to keep it the way it works for now, we'll iterate on this later. Works for you?